modules/javafx.controls/src/test/java/test/javafx/scene/control/skin/LabelSkinTest.java

```diff
@@ -2091,6 +2093,47 @@ public void testEscapedMnemonicSymbolIsNotProcessedWhenParsingIsDisabled() {
         assertEquals("foo __bar", LabelSkinBaseShim.getText(label).getText());
     }
 
+    /*********************************************************************
+     *
+     * Tests for bug reports                                             *
+     *
+     ********************************************************************/
+
+    //Test for JDK-8172849
```

Comment:
minor: I would have used a javadoc comment instead of the big comment block
/** Test for JDK-8172849 */

Reply:
Yes javadoc command would look better. I used the comment block to keep it consistent with the other comments present in the same file. I think if changing, then better to change all the comment blocks in the file.
Please let me know your thoughts.

Reply:
Please don't change all of the other comment blocks as part of this PR. You can file a follow-up bug if you like, but in that case, we should first define a "best practice", probably something that would align with what the JDK's jtreg test harness expects. So I think this can be left as-is for now.

Reply:
Sure Kevin. I'll keep the comments as-is and file a follow-up bug for this.